nucypher/blockchain/eth/actors.py

```diff
@@ -1566,12 +1563,12 @@ def block_until_ready(self, poll_rate: int = None, timeout: int = None, feedback
             # Bonding
             if (not bonded) and (staking_address != NULL_ADDRESS):
                 bonded = True
-                emitter.message(f"✓ Worker is bonded to ({staking_address})!", color='green', bold=True)
+                emitter.message(f"✓ Worker is bonded to {staking_address}", color='green')
 
             # Balance
             if ether_balance and (not funded):
                 funded, balance = True, Web3.fromWei(ether_balance, 'ether')
-                emitter.message(f"✓ Worker is funded with {balance} ETH!", color='green', bold=True)
+                emitter.message(f"✓ Worker is funded with {balance} ETH", color='green')
```

Comment:
> Jake: What?
Elaine: Oh it's nothing.
Jake: What is it?
Elaine: It's nothing.
Jake: Tell me.
Elaine: Well, I was just curious why you didn't use an exclamation point?